### DISCUSSION TITLE
Prevent signature window from overlapping with the current line

### DISCUSSION THREAD

**Discussion by madflow** [folke/noice.nvim#872] [Q&A]
"Quite often" I run into the issue, that the signature window is overlapping with the current line I am editing. In this case I am basically typing blindly when the window is open. For illustration I have forced this issue with this example:

![Kapture 2024-06-23 at 15 50 50](https://github.com/folke/noice.nvim/assets/183248/1d14444d-c190-477b-8590-f95438123203)

In the first line the window and line overlap. When I just push down the line once, everything is peachy again.

Is there a way I can fine-tune this behaviour with configuration?

I am using Lazyvim with this basic config:

```lua
return {
  {
    "folke/noice.nvim",
    opts = {
      presets = {
        lsp_doc_border = true,
      },
      lsp = {
        signature = {
          opts = {},
        },
      },
    },
  },
}
```

**Comment by ralvescosta** [comment]
I am using NvChad, and this configuration works for me

```
opts = {
    lsp = {
         signature = {
               enabled = false
         }
    }
}
```
